blockstack/blockstack-android
# Add listfiles API method

This PR
* adds method `Blockstack.listFiles` with documentation and integration tests.
* separates integration tests of Blockstack session into different files 
* adds also method `Scope.fromJSName` to fix #72 

Steps to test  `listFiles`
1.  clone feature/listfiles branch
1. open in Android studio
1. run example app
1. use last button `list files` to see a list of files

Steps to test `fromJSName`
1. run unit tests
![device-2018-11-05-113924](https://user-images.githubusercontent.com/1449049/47993340-83edb780-e0ef-11e8-95f3-c86d149736b1.png)
